RFC 0072: Legal Team (#72)
QuiltMC/rfcs: structure/0072-legal-team.md

> # RFC 0072: Legal Team

Reviewer: On a point of formatting: all headers within this document other than this title header should be increased by one level, as there should only be one H1 header (this one).

Author: Hard disagree - it makes the file much more navigable by only going from h1 to h3. It doesn't make sense to only have h2.

Reviewer: Can you please clarify for me what you mean? Are you saying that it is more navigable to have multiple H1 headers with a single nested level of H2 headers, compared to an H1 header for the title, H2 headers for each section, and a nested level of H3 headers (under the Explanation section)?

Based on a survey of all current RFCs, 9 out of 16 RFCs use H2 headers for the RFC sections, with 8 of those H2-using RFCs (and 1 of H1-using RFCs) using the H1 header as the title of the RFC.

If we expand the survey to all merged _and_ proposed RFCs, then 16 out of 26 use H2 headers, with 11 of these using H1 for the RFC title. The RFC template also uses H2 headers, with a single H1 header.

I therefore believe that, for better consistency with the majority of the current and proposed RFCs, the headings here should be modified so only one H1 header remains as the title, with sub-headings using H2 and lower-ranked headings.

This document has been written according to the process outlined in [RFC 0006](https://github.com/QuiltMC/rfcs/blob/main/structure/0006-governance.md#teams).

# Summary

The Legal Team is a standalone team in charge of handling legal matters to the best of its ability. This team will not require its members to be lawyers 

> or have any form of training, although this would be preferred. 

Reviewer: On a point of formatting: some lines within this document have trailing spaces. I believe those should be removed.

# RFC 4269: Legal Team

This document has been made according to the process outlined in [RFC 0006](https://github.com/QuiltMC/rfcs/blob/main/structure/0006-governance.md#teams).

# Summary

The Legal Team is a standalone team in charge of handling legal matters to the best of its ability. This team will not require its members to be lawyers 
or have any form of training, although this would be preferred. 

# Motivation

> Historically, legal matters have been handled by various different people without any proper structure. The de-factor team who would handle particular 

Reviewer: Could also be written as `*de facto*` ("The *de facto* team ..."), without the hyphenation and (optionally) italicized.

matters (eg. the Admin Board for registering Quilt as a non-profit) can feel unqualified to handle those issues and/or not have the bandwidth for it. 
Moreover, someone handling legal matters leaving that de-facto team should not lead to them not handling those matters anymore, especially 
considering the current scarcity of trained or knowledgeable members.

Some laws may also require us to have a designated legal team, such as to designate a Data Protection Officer 
([Art. 37 EU GDPR](https://gdpr-info.eu/art-37-gdpr/)).

# Explanation

## Responsibilities

The team is responsible for the following:

> - Ensuring Quilt stays up to date with data regulation laws (eg. EU GDPR, COPPA)

Author: That's the official name of the regulation

>     - Keep the privacy policies, and terms of services up to date

Reviewer 2: No comma needed here.

    - Designate a DPO
    - Responding to data privacy requests
    - Assisting the Infrastructure team in staying compliant
- Reviewing contracts signed by the organisation
- Handling registration of the organisation as a legal entity, if decided
- Advising other teams on any legal issues that may arise
- Answering questions and concerns from community members to the best of their ability
- Writing guidelines on what changes to Quilt projects require legal verification and providing a process for this verification to happen

> - Write internal or public processes on how to handle requests

Reviewer 2: Bad tense here again

- Any other legal matter not explicitly listed in this document

## Legal Team Server

> To perform their activity, the team has a private server not viewable by any other team, for discussing problems needing the uttermost privacy. 

Reviewer 3: You claim in a comment that this server is meant to be used only when neccesary, but this sentence states that it exists "to perform their activity" generally. I recommend something like this:

```suggestion
The Legal Team has a private server not viewable by any other team. This server may only be used for discussing problems which need the uttermost privacy. 
```

While this deviates from the usual observable approach that makes sure every team can be monitored by at least one other independent team, 

> privacy is legally required to handle some matters, such as data privacy requests.

Reviewer 2: We should discuss how we regulate this and assign users to it. Maybe this is also an infra team job? Potentially we need a separate infra team account for this? Or, perhaps, this should still be keyholder-relevant?

That discussion should be detailed here.

Alternatively, if that is not possible, we should discuss outsourcing this team to another organization somehow.

Author: We cannot legally justify having more people than strictly necessary seeing sensitive data. An overseeing body isn't strictly necessary according to the law. Usage of that server will be limited to only when legally required.

## Requesting Changes

In order to stay compliant with regulations and laws, the Legal Team has the power to block any Pull Request or community space change that
could negatively affects compliance. This overrides any decision made by a Team Lead.

> If a conflict arises between the Legal Team and a Team Lead, the Admin Board can be summoned for arbitration. In arbitration, the 

Reviewer 3: What does the removal of staff process look like for the legal team? Can we call a vote to remove someone if they overstep their bounds?

Author: That's defined in RFC 6

> Admin Board's decision is final and overrides the Legal Team.

Author: There is an odd number of admins to avoid that

The Legal Team also has the power to open high-priority issues on organisation repositories and request changes to be 
made as fast as reasonably possible.

## Communication

To be able to communicate with users, Legal Team members are provided with an email address, connected to one or many distribution lists.

## Transparency Reports

Because of the lack of oversight, the Legal Team will be required to publish public reports twice as year, containing as many information on

> their activity as allowed by the applicable laws, and taking into account user privacy.

Author: Taking into account the amount of legal requests we received so far, a report over 3 months of activity will most likely have zeros all across the board. I guess if we always use the same document template that's not too big of an issue. I feel like this set of guidelines is beside the point of this RFC, though. The only real obligation is what is already written down, having some flexibility over the structure and the format would be nice, as long as we keep the same level of transparency.

# Drawbacks

This team will most likely contain non-professional members. While their knowledge may still be extensive of some legal issues, we have to keep in 
mind they aren’t lawyers.

Due to legal needs, some if not all of the team members may have their real names appear publicly. Explicit approval from a team member will be 

> required before publishing.

Author: I'm not too sure what to say there. For example, if Quilt is registered as a non-profit, all team members would have to sign the bylaws, forcing their name to become public. There isn't much we can do about it other than make sure everyone is OK with that.

    - Responding to data privacy requests
    - Assisting the Infrastructure team in staying compliant
- Reviewing contracts signed by the organisation
- Handling registration of the organisation as a legal entity, if decided
- Advising other teams on any legal issues that may arise
- Answering questions and concerns from community members to the best of their ability
- Writing guidelines on what changes to Quilt projects require legal verification and providing a process for this verification to happen
- Writing internal or public processes on how to handle requests
- Any other legal matter not explicitly listed in this document

## Legal Team Server

The Legal Team has a private server not viewable by any other team. This server must only be used for discussing problems 
which need the uttermost privacy, as required by law. This includes, for example, talking about an ongoing data request or DMCA takedown
without any name being redacted.

While this deviates from the usual observable approach that makes sure every team can be monitored by at least one other independent team, 
privacy is legally required to handle some matters, such as data privacy requests.

> The server shall be owned by one of the Legal Team leads, who is in charge of managing the server. 

Reviewer: Given that this precludes oversight of the Legal Team's decisions or actions by other teams/organization members, should there be a required (or at least highly recommended) minimum of two members in the Legal Team, such that there is some level of accountability with respect to each other?

Reviewer: I'm thinking this from the angle of accountability and mutual oversight: having/recommending/requiring another person on the Legal Team means that:
 - If the first person is absent or otherwise preoccupied, there is another who can act as well;
 - Information is known to both (as much as applicable laws allow) rather than sequestered only to a single person;
 - Each member can [over]see the actions of the other, and give their recommendations/opinions whilst retaining privacy of the matter to within the Legal Team.

It might be too much to some, but I think there is some value to requiring mutual oversight within the team, since the Legal Team is otherwise unchecked in discussions/decisions by other teams (outside of the required transparency reports and Administrative Board arbitration).

Author: I can add it as a recommendation if you really want, but that cannot be a strict rule. The legal team still has to function even if there is a single member in it.

Reviewer: Fair enough then.

could negatively affect compliance. This overrides any decision made by a Team Lead.

If a conflict arises between the Legal Team and a Team Lead, the Admin Board can be summoned for arbitration. In arbitration, the 
Admin Board's decision is final and overrides the Legal Team.

The Legal Team also has the power to open high-priority issues on organisation repositories and request changes to be 
made as fast as reasonably possible.

## Communication

To be able to communicate with users, Legal Team members are provided with an email address, connected to one or many distribution lists.

## Transparency Reports

Because of the lack of oversight, the Legal Team will be required to publish public reports twice as year, containing as many information on
their activity as allowed by the applicable laws, and taking into account user privacy, similar to [Discord's transparency reports](https://discord.com/tags/transparency-reports).

# Drawbacks

This team will most likely contain non-professional members. While their knowledge may still be extensive of some legal issues, we have to keep in 
mind they aren’t lawyers.

> While legal liability may be a concern, the non-profit and its insurance will be here to provide some protection if an issue arises.

Reviewer: > the non-profit and its insurance
Could you clarify this?
